pyvista/utilities/fileio.py

```diff
@@ -233,3 +235,58 @@ def read_exodus(filename,
 
     reader.Update()
     return pyvista.wrap(reader.GetOutput())
+
+
+def read_meshio(filename, **kwargs):
+    """Read any mesh file using meshio."""
+    # Import meshio
+    import meshio
```

Comment:
I would put this in a `try` statement in case it's not installed and warn the user accordingly to make it optional.